package.json

```diff
@@ -140,6 +142,7 @@
     "@reduxjs/toolkit": "^1.6.1",
     "@sentry/react": "^7.45.0",
     "@sentry/tracing": "^7.45.0",
+    "@sentry/types": "^7.45.0",
```

Comment:
`@sentry/types` is imported from as part of `src/tracing`, but it was "shadow-installed" through other sentry packages. `hardhat` uses an older version of sentry, so it was necessary to install it so that typechecking would not break - not to mention it shouldn't have been "shadow-installed" in the first place.